### PR TITLE
Add sapo

### PR DESCRIPTION
**Pull request by esteveste**

### CODE REVIEW

File: _data/email.yml [hunk 1]
```diff
@@ -167,6 +167,11 @@ websites:
       - software
     doc: https://help.runbox.com/account-security/
 
+  - name: Sapo
```
Comment (Carlgo11) [inline _data/email.yml]:
```suggestion
  - name: SAPO Mail
```

File: _data/email.yml [hunk 2]
```diff
@@ -167,6 +167,11 @@ websites:
...
+    url: https://mail.sapo.pt
+    email_address: apoioportalsapo@suporte.sapo.pt
+    img: sapo.png
+
```
Comment (Carlgo11) [inline _data/email.yml:177]:
```suggestion
    twitter: sapo
    facebook: sapo
    lang: pt

```

File: _data/email.yml [hunk 3]
```diff
@@ -167,6 +167,11 @@ websites:
       - software
     doc: https://help.runbox.com/account-security/
 
+  - name: Sapo
+    url: https://mail.sapo.pt
+    email_address: apoioportalsapo@suporte.sapo.pt
```
Comment (Carlgo11) [inline _data/email.yml]:
```suggestion
    email_address: conteudos@mail.sapo.pt
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Try this logo instead

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
Thanks for the pull request! We just need one more member to approve it before we can merge.
By the way, we're missing a Portuguese translation of our Twitter messages so if that's something you'd like to help with we'd greatly appreciate that 😄 
https://github.com/2factorauth/twofactorauth/blob/master/_data/languages.yml

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**